Formula/mmctl.rb

```diff
@@ -0,0 +1,29 @@
+class Mmctl < Formula
+  desc "Remote CLI tool for Mattermost server"
+  homepage "https://github.com/mattermost/mmctl"
+  url "https://github.com/mattermost/mmctl.git",
+      :tag      => "0.2.1",
+      :revision => "c330b36f679b69ac33fa1e561190ad2eb3466777"
+  head "https://github.com/mattermost/mmctl.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOBIN"] = buildpath/bin
+    ENV["ADVANCED_VET"] = "FALSE"
+    ENV["BUILD_HASH"] = Utils.popen_read("git rev-parse HEAD").chomp
+    ENV["BUILD_VERSION"] = version.to_s
```

Comment:
FWIW, it probably makes more sense to set make parameters in the call to make rather than as environment variables, with something like this:
```ruby
ENV["GOBIN"] = buildpath/"bin"

hash = Utils.popen_read("git rev-parse HEAD").chomp
(buildpath/"src/github.com/mattermost/mmctl").install buildpath.children
cd "src/github.com/mattermost/mmctl" do
  system "make", "install", "ADVANCED_VET=FALSE", "BUILD_HASH=#{hash}", "BUILD_VERSION=#{version}"
end
```

Reply:
Furthermore, you don't need to set `BUILD_HASH` here because the makefile will generate the same result if you don't specify a value.

Reply:
I'd rather prefer to pass them as environment variables instead of making a huge command because it's only calling one command so it's not going to create a problem there.

And I prefer to get the BUILD_HASH explicitly even though the Makefile does it, because the person that reads the code in the future knows where is being set and where that comes from.